### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ploth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,8 +27,7 @@ this is not for you. For such devices, see
     VMs
   * My dumped VBIOS did **not** have EFI support, but I don't think that is
     necessary when the card has no outputs. rom-parser output for reference:
-    ```
-Valid ROM signature found @0h, PCIR offset 190h
+    ```Valid ROM signature found @0h, PCIR offset 190h
```
Comment (coderobe) [inline README.md]: This produces the wrong output. Appending a line to the triple-backticks sets a syntax-highlighting and won't show the text.

**Review by coderobe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ploth) [reply README.md]: Currently it looks like this:
image
And with the back ticks on the same row:
image

Comment (coderobe) [reply README.md]: Correct. Now compare both: Your amendments swallow the first line of the output

Comment (ploth) [reply README.md]: Funny :v:

**Pull request closed without merging**